proxy/ProxyClientTransaction.h

```diff
@@ -211,7 +211,11 @@ class ProxyClientTransaction : public VConnection
 
   virtual bool allow_half_open() const = 0;
 
-  virtual const char *get_protocol_string() const = 0;
+  virtual const char *
```

Comment:
Yeah, it's better to reduce strlen calls. I start thinking about returning int or enum from here (something like `TS_SSN_PROTOCOL_INDEX_HTTP_x_x`) and convert it to string in logging phase.
Do we really need to return string at here?